frontend/src/components/search/SearchVersionTwo.tsx

```diff
@@ -47,7 +47,7 @@ export function SearchVersionTwo({
       <QueryProvider>
         <div className="grid-container">
           <div className="desktop:display-flex desktop:margin-bottom-2">
-            <div className="flex-6">
+            <div className="flex-6 flex-align-self-end">
```

Comment:
unrelated styling fix to correctly align the filter drawer button when logged in